### DISCUSSION TITLE
Don't Include Annotations in Y-Axis Auto Scale

### DISCUSSION THREAD

**Discussion by stephenpapierski** [chartjs/chartjs-plugin-annotation#915] [Q&A] [answered]
I have a situation where I'm using a line annotation to denote a threshold value. That means that often times, the line might be significantly higher on the y-axis than the actual data. I'd like to exclude the line from the auto scaling of the y-axis and allow it to be off the chart. I know I can set my own min/max, but I would love to use the auto scaling, but ignore the annotation. Is that at all possible? Maybe with callbacks somehow?
### Result when line value is higher than data max
<img width="760" alt="image" src="https://github.com/chartjs/chartjs-plugin-annotation/assets/5900005/4d5ff77a-c9d1-47dd-b592-85548fcefdf7">

### Desired result when line value is higher than data max
<img width="776" alt="image" src="https://github.com/chartjs/chartjs-plugin-annotation/assets/5900005/4030cae1-6cc9-4baa-a2a1-8e95db07b93f">

### Desired result when line value is within data max
<img width="774" alt="image" src="https://github.com/chartjs/chartjs-plugin-annotation/assets/5900005/889c5d62-9630-4efd-bd28-1e0cc94be7b8">

**Comment by stockiNail** [comment, marked as answer]
@stephenpapierski the `adjustScaleRange` option should manage this behaviour. By default is set to `true`, therefore the plugin change the min/max of the chart to show always the annotation. 
You should try setting the  `adjustScaleRange` to `false`.
Let me know if works.

**Reply by stephenpapierski** [reply]
That does the trick. I swear I read the documentation, not sure how I missed that. Thanks for the gracious response.
